app/controllers/users_controller.rb

```diff
@@ -0,0 +1,32 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: %i(show)
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t(".success")
```

Comment:
```suggestion
      flash[:success] = t ".success"
```
chỗ nào bỏ dc ngoặc thì bỏ